pysipp/agent.py

```diff
@@ -39,7 +39,7 @@ class UserAgent(command.SippCmd):
     configuration options for a SIP UA.
     """
     # we skip `error` since we can get it from stderr
-    _log_types = 'screen calldebug message log'.split()
+    _log_types = 'screen calldebug message log error'.split()
```

Comment:
@fchareyron So we originally had this actually but found that `SIPp` wouldn't always generate it.
You can see from the failed `Travis` jobs that this is still true.
I'd rather see reliable behavior wrapped by `pysipp` and the unit test suite running clean.